api/src/ginger_calls.rs

```diff
@@ -200,6 +200,9 @@ pub fn read_field_element_from_buffer_with_padding(buffer: &[u8]) -> IoResult<Fi
     new_buffer.extend_from_slice(buffer);
     for _ in buff_len..FIELD_SIZE { new_buffer.push(0u8) } //Add padding zeros to reach field size
 
+    //Truncate buffer to fit the MODULUS
```

Comment:
I wouldn't put this in the code Rust side: it's the caller Java side who should be responsible of passing a correct byte array

Reply:
Done